packages/selected-network-controller/tsconfig.build.json

```diff
@@ -8,7 +8,8 @@
   "references": [
     { "path": "../base-controller/tsconfig.build.json" },
     { "path": "../network-controller/tsconfig.build.json" },
-    { "path": "../json-rpc-engine/tsconfig.build.json" }
+    { "path": "../json-rpc-engine/tsconfig.build.json" },
+    { "path": "../permission-controller/tsconfig.build.json" }
```

Comment:
A few leftover changes here from when the permission controller had been a dependency

Reply:
removed here: https://github.com/MetaMask/core/pull/3908/commits/6c3054e6df0218f1ef40fe6d394bc5287168b976